### ISSUE TITLE
￼Encoding in Style: a StyleGAN Encoder for Image-to-Image Translation

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#40]
# TL;DR
 Image2image translation that convert an input image into a latent representation (style representation), then put into the generator of the pretrained styleGAN. Since the input is converted into a latent representation, it is possible to transform the image without being bound by the pixel information of the input image (e.g., changing the direction of the face to the front). In addition, this framework can be used to perform a wide range of tasks such as pix2pix (e.g., high-resolution, face orientation transformation).
![image](https://user-images.githubusercontent.com/24524018/89900558-dd690b80-dc1e-11ea-8d1a-dde6d6c214f6.png)

# Why it matters:

# Paper URL

https://arxiv.org/abs/2008.00951

# Submission Dates(yyyy/mm/dd)
2020/08/03

# Authors and institutions
 Elad Richardson, Yuval Alaluf, Or Patashnik, Yotam Nitzan, Yaniv Azar, Stav Shapiro, Daniel Cohen-Or
- Penta-AI 
- Tel-Aviv University 

# Methods
The network structure is based on Feature Pyramid Networks, which are often used in object detection, to obtain a style representation.
They do not use adversarial learning, but train three types of learning objectives ; L2 distance per pixel, perceptual loss (e.b. F is pre-trained VGG), and identity loss using ArcFace (R in the following formula).
![image](https://user-images.githubusercontent.com/24524018/89900595-e8bc3700-dc1e-11ea-888d-418049138ffa.png)
![image](https://user-images.githubusercontent.com/24524018/89900606-ec4fbe00-dc1e-11ea-8f8f-cb347407a226.png)
![image](https://user-images.githubusercontent.com/24524018/89900619-eeb21800-dc1e-11ea-8b48-25489aad16d8.png)
![image](https://user-images.githubusercontent.com/24524018/89900626-f07bdb80-dc1e-11ea-8c38-6ee2232f58b6.png)
![image](https://user-images.githubusercontent.com/24524018/89900631-f2459f00-dc1e-11ea-92b3-63ec40ccee9b.png)

# Results
is framework can be used to perform a wide range of tasks such as pix2pix (e.g., high-resolution, face orientation transformation).

# Comments
